### PR TITLE
Metadata popup too big on mobile

### PR DESCRIPTION
**Pull request by RenataMuellerC2C**

### CODE REVIEW

**Review by ger-benjamin — APPROVED** [review]
Thanks.
It looks good for the popup from osm_open, but not so good for Theme "edition"-> layer "point" (click on the `i` of the layer in the layertree to open the popup). This popup is too small.
Can you take a look ? but if that's looks too complicated to change, perhaps you have to speak with the team to know if it's worth it to fix it.

File: contribs/gmf/src/controllers/AbstractMobileController.js [hunk 1]
```diff
@@ -79,6 +79,16 @@ export class AbstractMobileController extends AbstractAppController {
...
 
+    /**
+     * @type {?string}
+     */
+    this.displaywindowHeight = '100vh';
+
+    /**
+     * @type {?string}
+     */
+    this.displaywindowWidth = '100vh';
```
Comment (ger-benjamin) [inline contribs/gmf/src/controllers/AbstractMobileController.js:90]:
```suggestion
    this.displaywindowWidth = '100vw';
```
is better no ?

**Review by ger-benjamin — APPROVED** [review]
One comment, otherwise perfect it's now

File: contribs/gmf/src/controllers/AbstractMobileController.js (hunk 1 above)
Comment (sbrunner) [reply contribs/gmf/src/controllers/AbstractMobileController.js:90]: +1

**Pull request merged**